bin/node-sass

```diff
@@ -204,6 +232,15 @@ function run(options, emitter) {
     options.includePath = [options.includePath];
   }
 
+  if (options.directory) {
+    if (!options.output) {
+      emitter.emit('error', 'Specify an output directory when using multi-file compilation');
+    }
+    if (!isDirectory(options.output)) {
+      emitter.emit('error', util.format('Multi-file compilation: requires destination %s to be a directory.', options.output));
+    }
```

Comment:
Can we add tests for these two conditions?
For example: [cli.js#L438-L448](https://github.com/sass/node-sass/blob/2d22d9c7129a8c45e261de9764efb5f2ee396c05/test/cli.js#L438-L448).